### ISSUE TITLE
Accessibility Issue - Character Limits not Conveyed in Product Title and Description Inputs

### ISSUE THREAD

**Issue by sherenemorris03** [DFE-Digital/find-information-products-services-frontend#121]
**Describe the bug**
Not in MVP
Issue 13 in the Accessibility report
The ‘Product title’ and ‘Short description’ input fields have min/max character attributes, but this information is not conveyed to users. Screen reader users are not notified when they reach the maximum character limit, which may lead to confusion or incomplete submissions.

User story:
As a screen reader user,
I want to be informed of character limits when typing into input fields,
So that I can avoid exceeding the limit and understand the constraints upfront.

**To Reproduce**
Steps to reproduce the behaviour:
1. Navigate to the Create New Product page.
2. Type into the ‘Product title’ or ‘Short description’ fields.
3. Observe that no hint text or character count feedback is provided.

**Expected behaviour**
Users should be informed of character limits before and during input, especially when nearing the maximum.

**Screenshots**
<img width="945" height="499" alt="Image" src="https://github.com/user-attachments/assets/755303e7-5490-48d1-9d86-b40ccfc2c485" />

**Additional context**
Provide the mix and max values required within the hint text for the inputs. A character count status could also be provided, particularly within the ‘short description’ input as it’s unlikely that users will hit the max for the product title.

Link to issue on Statement Service
https://accessibility-statements.education.gov.uk/services/DACFEE24-0602-46AD-AE7C-19A23DADA42D/issues/229764B1-AC7A-489A-AD45-DA62228311FD
